templates/sysdig-cloud/0/rancher-compose.yml

```diff
@@ -25,3 +25,8 @@
       type: "string"
       default: sysdig.exclude_sysdigcloud=true
       required: true
+    - variable: "SDC_ADDITIONAL_CONF"
+      label: "Additional Configuration"
+      description: "Content must be valid yaml code, use the escape \n parameter for new lines and adhere to proper indent requirements."
+      type: "string"
```

Comment:
Sorry, I did test this and it does need the new lines as \n and not the ascii-13 char.